prime-dotnet-webapi/ViewModels/SiteRegistration/OrganizationListViewModel.cs

```diff
@@ -41,6 +41,12 @@ public IEnumerable<string> MatchedOn(string textSearch)
                 matchedOn.Add(nameof(Name));
             }
 
+            if (SigningAuthority != null && (SigningAuthority.FirstName + " " + SigningAuthority.LastName).Contains(textSearch))
```

Comment:
without converting the Signing Authority's name to lower case this will likely never match

Reply:
Converted to lower